src/styles/auth.css

```diff
@@ -0,0 +1,167 @@
+.form-main {
+    display: flex;
+    margin-top: 7%;
+    align-items: center;
+    justify-content: center;
+}
+.sign{
+    margin-top: 2%;
+}
+.form-container {
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+    align-self: center;
+    display: flex;
+    padding: 1.25rem 2rem;
+    justify-content: center;
+    background-color: #1B262C;
+    border-radius: 5px;
+
+}
+.signup-form{
+    /* width: 300px; */
+    color: #a0b2b9;
```

Comment:
Try using color variables as this colour is used at multiple places.

Reply:
Oh yeah, forgot to do that, will change it.